.golangci.yaml

```
@@ -135,6 +135,16 @@ linters-settings:
     - unnecessaryDefer
     - whyNoLint
     - wrapperFunc
+  tagliatelle:
+    case:
```

We should probably add the "normal" rules here also
```suggestion
    case:
      rules:
        json: 'camel'
```

---

In the documentation it is said "Important: the `extended-rules` overrides `rules`" (https://golangci-lint.run/usage/linters/#tagliatelle) so I am not sure if they should be there if they are overridden

---

Well, we need something to override, some kind of base line. I think the way it is supposed to work is that we put the "default" that we want in the rules and then if it doesn't quite work for everything, we extend by overriding in the extended-rules.